code-challenges.js

```diff
@@ -37,9 +74,35 @@ const currentCohort = "Golf 2023"
 
 // Pseudo code:
 
+// Utilize input of a string. The string is currentCohort = "Golf 2023".
+// Will use .split method to convert the string into an array.
+// Will use .reverse method to reverse the order of the values in the array.
+// Will use .join method to convert the array values back into a string.
+// Will declare a variable that uses these methods on the original string.
+
+var revCohort = currentCohort.split("").reverse().join("")
+console.log(revCohort)
+
+// Output was "3202 floG".
+
 // --------------------3) Create the code that will log only the odd numbers from the array using the test variable provided below.
 
 const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
 // Expected output: 13 5 -5 27
 
 // Pseudo code:
+
+// Utilize input of an array. The array is stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10].
+// Make a loop to evaluate each of the values in the array.
+// Use modulo to return the remainder from each of the array values.
+// Use an equality operator to determine if the remainder is divisible by 2.
+// Use a bang operator to look for a value not divisible by 2.
```

Comment:
This pseudo code is a little confusing. The code isn't checking if the remainder is divisible by 2, it's checking if the current value in the iteration does not have a remainder of 0 when divided by 2.